public/app.css

```diff
@@ -0,0 +1,6 @@
+figure {
+    max-width: 16px !important;
+    margin-right: 12px;
```

Comment:
Good fix! Just have some questions.

Is this same as "$euiSizeM" in the related link you provided? Curious how you get the value. 
Also for max-width, looks like this is not in the related link. Did you notice it during your UI test?

Can you also perform a search of "figure" in the security plugin plugin code and Kibana code to see if it is used by other components? Just want to be cautious that it won't break other components accidentally.

Reply:
Yes it is same as "$euiSizeM" ([Link to CSS](https://github.com/elastic/eui/blob/454c73415cdba25685b7070ebd4bf847f39e6ebd/src/global_styling/variables/_size.scss#L5))
First I got this css class and values by inspecting element in developer tool. Then I verified the css in the elasticsearch/kibana repo. 

I have gone through the pages to make sure there's no other figure components used by security plugin code.